src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js

```diff
@@ -45,6 +53,7 @@ const MobileWorkingWorkOrdersView = ({ currentUser }) => {
       const data = await frontEndApiRequest({
         method: 'get',
         path: url,
+        timeout: 5000,
       })
```

Comment:
Add 5 second timeout to request, incase it hangs. If it takes that long, its better to fail